insights/parsers/os_release.py

```diff
@@ -67,3 +67,11 @@ def data(self):
             Deprecated, it will be removed from 3.7.0
         """
         return self
+
+    @property
+    def is_rhel_ai(self):
```

Comment:
There are also a combiner `OSRelease` from `insights/combiners/os_release.py`. 
Shall we also propagate this `is_rhel_ai` into the combiner?  

@xiangce any suggestion on this?

Reply:
As the `OSRelease` combiner is majorly for identify if the host an RHEL OS system.  
https://github.com/RedHatInsights/insights-core/blob/644d314cd574da23dd3d52d24f78bb8b4b84ddcd/insights/combiners/os_release.py#L4-L5
And the "is_rhel_ai" can completely being decided by the "Specs.os_release" alone.
It's not so necessary to add this interface to the Combiner, in my opinion.

But it's up to you, depends on how rule devs preference. I'd leave it to @JoySnow and @wushiqinlou

Reply:
@JoySnow - Let's keep monitoring. If it is necessary to add the func into combiner during the development in the future, I will add it.